src/main/scala/viper/server/frontends/http/jsonWriters/ViperIDEProtocol.scala

```diff
@@ -408,9 +408,9 @@ object ViperIDEProtocol extends akka.http.scaladsl.marshallers.sprayjson.SprayJs
   })
 
   implicit val jsonStreamingSupport: JsonEntityStreamingSupport = {
-    val start = ByteString("")
+    // val start = ByteString("")
```

Comment:
Could you explain these two comments, please?

Reply:
This was another unused variable. I guess it would be better to simply drop the statement

Reply:
I'm not so sure it's unused since it's inside an implicit object. IIRC this is needed in Json streaming over Http in order to prevent leading and trailing characters, e.g., we want a stream of the form 
```
{message}
{message}
{message}
```

rather than e.g.

```
[{message},
{message},
{message}]
```

Reply:
Interestingly I did not know this and apparently IntelliJ / the Scala compiler was not aware either. I'll switch to `Flow[ByteString].intersperse(start, between, end)` to make it more explicit